### PR TITLE
Update List of features for Godot 4.0.rc1

### PR DESCRIPTION
**Pull request by Calinou**
This also adds more links to documentation within the page.

### CODE REVIEW

File: about/list_of_features.rst [hunk 1]
```diff
@@ -78,32 +81,47 @@ Editor
...
    - Hard or soft shadows (adjustable on a per-light basis).
    - Custom shaders can access a real-time :abbr:`SDF (Signed Distance Field)`
-     representation of the 2D scene, which can be used for improved 2D lighting
-     effects including 2D global illumination.
+     representation of the 2D scene based on :ref:`class_LightOccluder2D` nodes,
+     which can be used for improved 2D lighting effects including 2D global illumination.
 
-- Font rendering using bitmaps, rasterization using FreeType or
-  multi-channel signed distance fields (MSDF).
+- :ref:`Font rendering <doc_using_fonts>` using bitmaps, rasterization using FreeType
```
Comment (Piralein) [inline about/list_of_features.rst:120]:
```suggestion
- :ref:`Font rendering <doc_gui_using_fonts>` using bitmaps, rasterization using FreeType
```

File: about/list_of_features.rst [hunk 2]
```diff
@@ -510,29 +536,30 @@ Input
 Navigation
 ^^^^^^^^^^
 
-- A* algorithm in 2D and 3D.
-- Navigation meshes with dynamic obstacle avoidance.
+- A* algorithm in :ref:`2D <class_AStar2D>` and :ref:`3D <class_AStar3D>`.
+- Navigation meshes with dynamic obstacle avoidance in
+  :ref:`2D <navigation_introduction_2d>` and :ref:`3D <doc_navigation_overview_3d>`.
```
Comment (Piralein) [inline about/list_of_features.rst:541]:
```suggestion
  :ref:`2D <doc_navigation_overview_2d>` and :ref:`3D <doc_navigation_overview_3d>`.
```

File: about/list_of_features.rst [hunk 3]
```diff
@@ -112,60 +130,57 @@ Vulkan renderer.
...
 - Perspective, orthographic and frustum-offset cameras.
-- When using the Vulkan Clustered backend (default on desktop), a depth prepass
-  is used to improve performance in complex scenes by reducing the cost of overdraw.
-
-- `OpenGL support planned for a future Godot 4.x release <https://godotengine.org/article/about-godot4-vulkan-gles3-and-gles2>`__.
-
-  - If you need OpenGL support, use Godot 3.x which remains supported.
+- When using the Forward+ backend, a depth prepass is used to improve
+  performance in complex scenes by reducing the cost of overdraw.
+- :ref:`doc_variable_rate_shading` on supported GPUs in Forward+ and Forward Mobile.
```
Comment (Piralein) [inline about/list_of_features.rst:178]: The reference `doc_variable_rate_shading` is invalid, not sure what to link here.

File: about/list_of_features.rst [hunk 4]
```diff
@@ -578,13 +605,14 @@ Windowing and OS integration
 Mobile
 ^^^^^^
 
-- In-app purchases on Android and iOS.
+- In-app purchases on :ref:`Android <doc_android_in_app_purchases>`
+  and :ref:`iOS <doc_plugins_for_ios>`_.
```
Comment (Piralein) [inline about/list_of_features.rst:609]:
```suggestion
  and :ref:`iOS <doc_plugins_for_ios>`.
```

File: about/list_of_features.rst (hunk 3 above)
Comment (Calinou) [reply about/list_of_features.rst:178]: This is pending on https://github.com/godotengine/godot-docs/pull/6402 to be merged :slightly_smiling_face:

**Review by YuriSizov — APPROVED** [review]
LGTM, thanks for proofreading it! Though we cannot merge until the referenced PR is merged, or until the text here is temporarily adjusted to remove the mention/link.

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**